### PR TITLE
Applied changes for Issue 3_a,b

### PR DESCRIPTION
**Pull request by navyasrim**
Issue 3 a,b
Dashboard → Click dot by person’s name in the Leaderboard 
This should show anyone who does this that person’s Dashboard. Right now nothing happens when you click the dot. Showing their complete Dashboard (exactly what you see for your own dashboard) is needed for anyone managing a person to be able to see/edit their action items that will be coming later, badges, etc.
There should also be some sort of popup that makes it clear to the viewer that they are viewing another person’s dashboard, not their own. The popup should say: “You are currently viewing the dashboard for XXXX. Click this popup to return to your own dashboard.” 

In  src/components/LeaderBoard/Leaderboard.jsx

Commented out the following code  
Line 188 :{/* <Link to={`/dashboard/${item.personId}`}> */}

Line 201 :  {/* </Link> */}

Added the below code in Line 187 
onClick={() => { if (window.confirm(`Are you sure you wish to view this ${item.name} dashboard ?`)) {window.open(`/dashboard/${item.personId}`, "Popup","toolbar=no, location=no, statusbar=no, menubar=no, scrollbars=1, resizable=0, width=580, height=600, top=30")}} }

just implemented on click event to show in another window.

### CODE REVIEW

**Comment by KitanGarcia** [comment]
Looks good! The code works and does its job. As a user it's easy to follow. That said, it does seem to stray a bit from the description. I think instead of the alert and new window you added, they want a pop up they want it to take place on the original page and with a message to return you to your original user rather than a warning/confirmation to view a different one. Since this is my first PR, I'm not sure how strict we are in details like this. If we're not, this is perfectly fine and functional! I would just say to feel free to remove the commented out code before calling it done!

**Comment by one-community** [comment]
That's correct Kitan, once this is merged and reviewed on the live site, I'm going to further update the Bug request. Normally I'd ask for a fix of what you said but this one is going need more work anyway, so I want to see it working as it is and then I'll request specific updates to improve it along with finishing the functionality part Navya said she didn't know how to do.

**Comment by one-community** [comment]
Please add your review Kitan and then approve this PR.

**Review by KitanGarcia — DISMISSED** [review]
Looks good! The code works and does its job. As a user it's easy to follow. That said, it does seem to stray a bit from the description. I think instead of the alert and new window you added, they want a pop up they want it to take place on the original page and with a message to return you to your original user rather than a warning/confirmation to view a different one. This is perfectly fine and functional. When we update it in the future to better match the description, let's remove the code comments.

File: src/components/LeaderBoard/Leaderboard.jsx [hunk 1]
```diff
@@ -184,8 +184,8 @@ const LeaderBoard = ({
             </tr>
             {leaderBoardData.map((item, key) => (
               <tr key={key}>
-                <td className="align-middle">
-                  <Link to={`/dashboard/${item.personId}`}>
+                <td className="align-middle" onClick={() => { if (window.confirm(`Are you sure you wish to view this ${item.name} dashboard ?`)) {window.open(`/dashboard/${item.personId}`, "Popup","toolbar=no, location=no, statusbar=no, menubar=no, scrollbars=1, resizable=0, width=580, height=600, top=30")}} }>
```
Comment (pranav1416) [inline src/components/LeaderBoard/Leaderboard.jsx:187]: You can write a function to be called onClick event for the table row, and insert the conditional logic in the function body. Also, please make use of [<Redirect>](https://v5.reactrouter.com/web/api/Redirect) from react-router to navigate to a different view. 'history' object is also an alternative, you can use history.push() to route the location of the user, but IMO it is preferable to route using react-router which is being used in the app.

**Review by pranav1416 — APPROVED** [review]

**Pull request merged**